src/SeleniumLibrary/utils/seleniumversion.py

```diff
@@ -0,0 +1,8 @@
+from collections import namedtuple
+
+import selenium
+
+SeleniumVersion = namedtuple('SeleniumVersion', 'major minor micro')
+version = selenium.__version__.split('.')
+selenium_version = SeleniumVersion(major=version[0], minor=version[1],
+                                   micro=version[2])
```

Comment:
- Copyrights missing from the file.
- +1 for namedtuple.
- Like you commented elsewhere, SELENIUM_VERSION is better name for the constant.
- Can we be certain that version always has three components? If not, this could be used:
   ```python
   major, minor, micro = (selenium.__version__.split('.') + ['0', '0'])[:3]
   SELENIUM_VERSION = SeleniumVersion(major, minor, micro)
   ```
   If earlier releases have always contained three components, the above is an overkill.

Reply:
Looked releases all to they to the 2.50.0 and there always has been three components, in the version field. But in one day, there might not be, so why risk it.

Others done.